WordPress/Classes/Networking/ReaderPostServiceRemote.m

```diff
@@ -287,6 +307,13 @@ - (RemoteReaderPost *)formatPostDictionary:(NSDictionary *)dict
     post.wordCount = [dict numberForKey:PostRESTKeyWordCount];
     post.readingTime = [self readingTimeForWordCount:post.wordCount];
 
+    NSDictionary *railcar = [dict dictionaryForKey:PostRESTKeyRailcar];
+    if (railcar) {
+        NSError *error;
+        NSData *railcarData = [NSJSONSerialization dataWithJSONObject:railcar options:NSJSONWritingPrettyPrinted error:&error];
```

Comment:
I thought it might make inspecting the stored string nicer.  I'm fine to drop it if you think we should.